sdk/storage/Azure.Storage.DataMovement.Blobs/stress/src/src/DataMovementBlobStressConstants.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Storage.DataMovement.Blobs.Stress
+{
+    internal class DataMovementBlobStressConstants
+    {
+        public const int KB = 1024;
+        public const int MB = KB * 1024;
+        public const int GB = MB * 1024;
+        public const long TB = GB * 1024L;
+        public const int Base16 = 16;
+
+        public static partial class EnvironmentVariables
+        {
+            // Shared Resources
+
+            /// <summary>
+            ///   The name of the environment variable that holds the instrumentation key for the
+            ///   Application Insights resource for the test runs.
+            /// </summary>
+            ///
+            public const string ApplicationInsightsKey = "APPINSIGHTS_INSTRUMENTATIONKEY";
+
+            /// <summary>
+            ///   The name of the environment variable that holds the connection string for the
+            ///   Event Hubs Namespace resource for the test runs.
+            /// </summary>
+            ///
+            public const string StorageBlobEndpoint = "STRESS_STORAGE_BLOB_ENDPOINT";
```

Comment:
I wasn't sure where to put this question, so I'll put it here. What auth does these tests use? Looks from the test setup its all OAuth?

Reply:
It's all OAuth, but shouldn't it be OAuth?

Reply:
Yep, OAuth is good. Just cruious.